worker/update_device_status.json

```diff
@@ -28,7 +28,7 @@
                 "${workflow.input.azure_status}"
             ],
             "http": {
-                "uri": "http://${env.AZURE_IOT_MANAGER_ADDR|mender-azure-iot-manager:8080}/api/internal/v1/azure-iot-manager/tenants/${workflow.input.tenant_id}/bulk/devices/status",
+                "uri": "http://${env.IOT_MANAGER_ADDR|mender-iot-manager:8080}/api/internal/v1/iot-manager/tenants/${workflow.input.tenant_id}/bulk/devices/status",
```

Comment:
the name of the workflow is still "update azure status" -- if it is on purpose, then ignore that comment.

Reply:
Yes, I think the name should stay "update azure status" as also the parameters are `azure_*`.
@alfrunes will change it once we start implementing additional cloud providers, I suppose.